lib/node_modules/@stdlib/array/typed-complex/docs/types/index.d.ts

```diff
@@ -22,7 +22,7 @@
 
 /// <reference types="@stdlib/types"/>
 
-import { ArrayLike, ComplexTypedArray, ComplexFloatingPointDataType as DataType } from '@stdlib/types/array';
+import { ArrayLike, ComplexTypedArray, ComplexArrayMap } from '@stdlib/types/array';
```

Comment:
I'd use `ComplexFloatingPointDataTypeMap` (or whatever the `*DataType` convention is) in order to be consistent. I'd rather be consistent and verbose, than inconsistent and unpredictable.

Reply:
Yes, definitely wanted to be consistent and settle on one naming scheme. `ComplexFloatingPointDataTypeMap` etc. sound good to me!

Reply:
This file still uses `ComplexArrayMap` which does not exist.

Reply:
Resolved.